BZ-13813: feat: Added stepper module

sandeepsj opened the pull request:
- Added Stepper properties type and its default value

prajwal-jp commented on src/lib/Stepper/properties.ts:
```diff
@@ -0,0 +1,24 @@
+};
+
+export type StepProperties = {
+  stepIndex: number;
```
Do we need a seperate index?

sandeepsj replied:
this index will be shown in the UI

iamHrithikRaj commented:
LGTM

sandeepssj committed:
BZ-13813: feat: Added stepper module properties

- Added Stepper module
- Added stepper properties type and
- Added Step component, that will be used by stepper, which shows each steps.
- imported all the above in lib/index.ts
